src/bootstrap/config.rs

```diff
@@ -617,6 +621,9 @@ impl Config {
         set(&mut config.initial_rustc, build.rustc.map(PathBuf::from));
         set(&mut config.initial_cargo, build.cargo.map(PathBuf::from));
 
+        let default = false;
+        config.llvm_skip_rebuild = llvm_skip_rebuild.unwrap_or(default);
```

Comment:
Please inline `default` here. I'm not sure why LLVM asserts below have it pulled out (no need to change that in this PR).

Reply:
Yeah, I also thought this was a bit odd, but decided to stick to the pattern that was there. Will fix this now. Thanks! 👍